app/jobs/sync_salesforce_program_to_lms_job.rb

```
@@ -10,6 +10,7 @@ def perform(program_id, email)
   end
 
   rescue_from(StandardError) do |_exception|
+    Rails.logger.error(_exception)
```

don't hide exceptions that happen during the job

---

Send to honeycomb too? Honestly, I think we should send the email and then re-raise letting it bubble up normally

---

I added a honeycomb line, but raising inside a rescue_from doesn't seem to be supported behavior. see https://simonecarletti.com/blog/2009/11/re-raise-a-ruby-exception-in-a-rails-rescue_from-statement/